ghga_event_schemas/pydantic_.py

```diff
@@ -301,6 +301,30 @@ class Config:
         title = "file_download_served"
 
 
+class EmailNotification(BaseModel):
+    """This event is emitted by services that desire to send an email notification.
+    It is picked up by the notification service."""
```

Comment:
```suggestion
class Notification(BaseModel):
    """This event is emitted by services that desire to send a notification e.g. via email.
    It is picked up by the notification service."""
```

Reply:
If we were ever to implement other channels of notification, the event structure would have to be changed. Like if we added SMS support we'd need a telephone number field. Then things like the subject would not be applicable and we'd need to check different fields in the event payload to figure out if we're supposed to send an email, a text, or both, and we'd lose that control provided by the required field. 
Please let me know what you think.